CMakeLists.txt

```diff
@@ -544,6 +545,11 @@ if(ENABLE_ANALYTICS)
   add_definitions(-DUSE_ANALYTICS=1)
 endif()
 
+if(ENABLE_XCB)
```

Comment:
Move this to the Vulkan backend CMake file and set the define on the target instead of globally.